### PR TITLE
Fixed warrior spell, Commanding Shout. Ref #894

### PR DESCRIPTION
**Pull request by Razfury**
Fixes warrior spell Commanding Shout for ref #894 includes the sql
Fixes 2 warlock spells as well.

This is my first pull request 😮 !

### CODE REVIEW

File: src/server/scripts/Spells/spell_warlock.cpp [hunk 1]
```diff
@@ -447,8 +451,11 @@ class spell_warl_shadow_bolt : public SpellScript
 
     void HandleOnCast()
     {
-        Unit* caster = GetCaster();
+		Unit* caster = GetCaster();
         caster->CastSpell(caster, SPELL_WARLOCK_SHADOW_BOLT_SHOULSHARD, true);
+
+		if (caster->HasAura(SPELL_WARLOCK_SHADOWY_INSPIRATION))
```
Comment (Traesh) [inline src/server/scripts/Spells/spell_warlock.cpp]: Please use 4 spaces instead of tabs

File: src/server/scripts/Spells/spell_warlock.cpp [hunk 2]
```diff
@@ -239,9 +241,11 @@ class spell_warl_call_dreadstalker : public SpellScript
         if (!caster)
             return;
 
-        if (caster->HasAura(SPELL_WARLOCK_IMPROVED_DREADSTALKERS))
-            for (int32 i = 0; i < 2; i++)
-                caster->CastSpell(caster, SPELL_WARLOCK_HAND_OF_GULDAN_SUMMON, true);
+		if (caster->HasAura(SPELL_WARLOCK_IMPROVED_DREADSTALKERS))
+			for (int32 i = 0; i < 2; i++)
```
Comment (Traesh) [inline src/server/scripts/Spells/spell_warlock.cpp]: Brackets here for the "if" (not needed for the else)

**Comment by Razfury** [comment]
Yes i Have @wonder2011

**Comment by wwolegion** [comment]
@Razfury not working. buff gets applied but hp increase doesn't work.

**Comment by wonder2011** [comment]
@wwolegion do you run the SQL from this PR ?

**Pull request merged**